### PR TITLE
BUG checkout properly and do not check solvable for branches

### PR DESCRIPTION
**Pull request by beckermr**
I think this should be the last of the bugs.

### CODE REVIEW

File: conda_forge_tick/auto_tick.py [hunk 1]
```diff
@@ -826,7 +835,7 @@ def _compute_time_per_migrator(mctx, migrators):
         else:
             num_nodes.append(len(mmctx.effective_graph.nodes))
 
-    num_nodes_tot = sum(num_nodes)
+    num_nodes_tot = max(sum(num_nodes), 1)
```
Comment (CJ-Wright) [inline conda_forge_tick/auto_tick.py:838]: Is this correct? Some migrations may have no nodes, like the migration migrator.

Comment (beckermr) [reply conda_forge_tick/auto_tick.py:838]: This is to prevent a divide by zero in the next line. I will move it there.

Comment (beckermr) [reply conda_forge_tick/auto_tick.py]: fixed

File: conda_forge_tick/auto_tick.py [hunk 2]
```diff
@@ -849,7 +859,7 @@ def _compute_time_per_migrator(mctx, migrators):
         time_per_migrator[i] = time_per_migrator[i] * time_fac
 
     # recompute the total here
-    tot_time_per_migrator = sum(time_per_migrator)
+    tot_time_per_migrator = max(sum(time_per_migrator), 1)
```
Comment (CJ-Wright) [inline conda_forge_tick/auto_tick.py:862]: Can this also be zero?

**Comment by CJ-Wright** [comment]
Also why no solve check for branches?

File: conda_forge_tick/auto_tick.py (hunk 2 above)
Comment (beckermr) [reply conda_forge_tick/auto_tick.py:862]: Yes. We divide by it later. It is only used for generating output IIRC. I'll move this call too.

**Comment by beckermr** [comment]
> Also why no solve check for branches?

We cannot report the errors properly on the status page, so it makes more sense to let the PRs through. Otherwise, people will not get branch PRs and will have to dig into the bot to figure out why.

This goes in the general direction of keeping branches as second class citizens until we can do a more proper job.

**Comment by beckermr** [comment]
Any more comments @CJ-Wright?

**Pull request merged**